autogen/agentchat/utils.py

```diff
@@ -77,7 +77,7 @@ def aggregate_summary(usage_summary: Dict[str, Any], agent_summary: Dict[str, An
             aggregate_summary(total_usage_summary, agent.client.total_usage_summary)
             aggregate_summary(actual_usage_summary, agent.client.actual_usage_summary)
 
-    return total_usage_summary, actual_usage_summary
+    return {"total_usage_summary": total_usage_summary, "actual_usage_summary": actual_usage_summary}
```

Comment:
Would it be more clear if we say "usage_including_cached_inference" and "usage_excluding_cached_inference" ?
cc @kevin666aa @qingyun-wu

Reply:
Yes, I think that is better. How about the names in class ChatHistory, should we use `ChatHistory.cost`?